src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/80.diff.sql

```diff
@@ -0,0 +1 @@
+Execute dbo.DeleteHistory 0,0,0
```

Comment:
What is the reason for this statement?

Reply:
What has happened is that after Schema 76 the IsHistory column was removed from search parameter tables. This column hadn't been being used since v48, but for any customers who had data older than that there could be historical data still in those tables. That data was preventing current data from being returned in search requests because it was filling up the count of resources the search was looking for. 
This stored procedure was written as part of v48 to clean up this data, but it was never actually run. This change explicitly runs this stored proc to finally clean up this unneeded data. It does not delete the historical resources, just the search parameters connected to them.